### ISSUE TITLE
SUMMARY: Meets Expectations

### ISSUE THREAD

**Issue by jzhu01** [zelig270/prj-rev-bwfs-tea-cozy#5]
Nice job on this project - I really liked how you styled the nav, and how you structured your locations section. In terms of improvement, I recommend moving the opacity to your images, reducing repetitive code in your CSS, as well as adding a margin: 0 auto to your sections.

If you’re in the mood for a challenge, I would look into implementing a mobile version of this page! Here's a link to get you started: https://www.w3schools.com/css/css_rwd_mediaqueries.asp. Keep up the awesome work! ^_^

**Comment by zelig270** [comment]
Yeah- my CSS is a bit wordy. Trying to learn to not be so repetitive. Hoping it will come with time! Thank you!
